components/clientComponents/forms/Form/Form.tsx

```diff
@@ -261,6 +277,7 @@ interface FormProps {
   onSuccess: (id: string) => void;
   children?: (JSX.Element | undefined)[] | null;
   t: TFunction;
+  allowGrouping: boolean | undefined;
```

Comment:
allowGrouping should be optional per the lint error below it's not relevant for all forms